setup.py

```diff
@@ -0,0 +1,333 @@
+    package_directories = dict()
+    
+    if not shogun_preconditions_met():
+        shogun_bootstrap()
+    
+    package_directories[''] = shogun_python_packages_location
+    
+    return package_directories
+
+
+def shogun_data_files():
+    data_files = list()
+    libshogun_files = glob.glob(os.path.join(shogun_generated_install, 'lib/libshogun*'))
+    modshogun_so_destination = os.path.join('lib', python_package_path(shogun_python_packages_location))
+    modshogun_so_file = glob.glob(os.path.join(shogun_python_packages_location, '_modshogun.so'))[0]
+    
+    # appending data files
+    data_files.append(('lib', libshogun_files))
+    data_files.append((modshogun_so_destination, [modshogun_so_file]))
+    
+    if show_debug_information: 
+        print('Shogun Python package data files:')
+        for data_file_content in data_files:
+            print('|->[%s]' % data_file_content[0])
+            
+            for data_file in data_file_content[1]:
+                print('    |--> %s' % os.path.basename(data_file))
+         
+        return data_files
+
+
+# https://docs.python.org/2/distutils/apiref.html#creating-a-new-distutils-command
+class ShogunBuild(distutils.command.build.build):
+    user_options = distutils.command.build.build.user_options + [('cmake=', None, 'Specify CMake arguments.')]
+    build_base = ''
+    build_lib = ''
+    build_scripts = ''
+    plat_name = ''
+
+    def initialize_options(self):
+        self.cmake = None
+    
+    def finalize_options(self):
+        pass
+    
+    def run(self):
+        print('Running Package build')
+        
+        if not shogun_preconditions_met:
+            shogun_cmake(self.cmake)
+            shogun_make()
+            shogun_make_install()
+    
+    # Command.sub_commands
+
+
+class ShogunInstall(setuptools.command.install.install):
+    def run(self):
+        print('Running Package install')
+        
+        if shogun_preconditions():
+            self.do_egg_install()
+
+
+shogun_setup(
+    name = "shogun",
+    version = get_shogun_version(),
+    
+    description = 'The Shogun Machine Learning Toolbox',
+    long_description = readme,
+    url = 'http://www.shogun-toolbox.org/',
+    
+    author = 'Ariane Paola Gomes',
+    author_email = 'arianepaola@users.noreply.github.com',
+    
+    license = 'The GNU General Public License v3.0',
+    
+    # https://pypi.python.org/pypi?%3Aaction=list_classifiers
+    classifiers = [
+        'Development Status :: 5 - Production/Stable',
+        'Intended Audience :: Developers',
+        'Intended Audience :: Education',
+        'Intended Audience :: Information Technology',
+        'Intended Audience :: Science/Research',
+        'Topic :: Software Development :: Libraries',
+        # Python 2 and Python 3 support
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 3'
+    ],
+    
+    keywords = 'Large scale kernel methods and Support Vector Machines (SVM)',
```

Comment:
Can you put our updated description?
"Unified and efficient Machine Learning"

https://github.com/shogun-toolbox/shogun/wiki/Project-description